api/tests/unit/domain/models/CertificationIssueReportResolutionStrategies_test.js

```diff
@@ -1250,6 +1251,302 @@ describe('Unit | Domain | Models | CertificationIssueReportResolutionStrategies'
     });
   });
 
+  context('#NEUTRALIZE_OR_VALIDATE_IF_FOCUSED_CHALLENGE', function () {
```

Comment:
Quelle est la raison pour ne pas mentionner le nom de la fonction ?
```suggestion
  context('#neutralizeOrValidateIfFocusedChallengeStrategy', function () {
```

Reply:
J'ai repris la typo des autres tests du fichier